src/vulkan/image_descriptor.h

```diff
@@ -44,6 +45,8 @@ class ImageDescriptor : public BufferBackedDescriptor {
 
  private:
   std::unique_ptr<TransferImage> transfer_image_;
+  Sampler* amber_sampler_;
```

Comment:
nit: move the null ptr here and remove the default value.

Reply:
Ok, I've also added a setter for this so I don't have to specify the pointer for all image descriptor objects.